src/profiler.rs

```diff
@@ -0,0 +1,63 @@
+//! Game plugin for performance profiling tools.
+
+use crate::prelude::*;
+
+/// Installs profiler ui plugins
+pub fn game_plugin(game: &mut Game) {
+    game.sessions
+        .create(SessionNames::PROFILER)
+        .install_plugin(session_plugin);
+}
+
+/// Designate session that marks profiler's frame boundary.
+///
+/// Installing to game session is recommended, however this will not work if
+/// trying to profile outside of match or while game session is paused.
+pub fn install_frame_marker(session: &mut Session) {
+    session
+        .stages
+        .add_system_to_stage(CoreStage::First, mark_new_frame);
```

Comment:
We should probably install the `mark_new_frame` system as a [game system](https://fishfolk.github.io/bones/rustdoc/bones_lib/struct.Game.html#structfield.systems) so that it runs before any created sessions.